tap_mysql/discover_utils.py

```diff
@@ -215,6 +219,10 @@ def schema_for_column(column): # pylint: disable=too-many-branches
         result.type = ['null', 'string']
         result.format = 'binary'
 
+    elif data_type in SPATIAL_TYPES:
+        result.type = ['null', 'string']
```

Comment:
can we extract into `['null', 'object']` instead of string?

In that case if target supports more efficient data type than string, then it will be loaded into it automatically. For example target-snowflake can use `VARIANT` and target-postgres has `JSON` column type.